senza/manaus/route53.py

```diff
@@ -281,7 +281,7 @@ def to_alias(self,
         if self.alias_target is not None:
             # Record is already an Alias
```

Comment:
It's used to [create the new record](https://github.com/zalando-stups/senza/pull/363/files/65b8b208e114ad8827b13b490af9b3e4ea23247f#diff-9932ff860b637166805b1f2b3bfe9e46L300)